sharktank/sharktank/ops/default_impls.py

```diff
@@ -752,10 +752,13 @@ def to_default(tensor: Tensor, *args, **kwargs) -> PrimitiveTensor:
 
 @trace_tensor.override(AllOfExprsVariadic(IsOfType(Tensor, InferenceTensor)))
 def trace_tensor(key: str, *tensors: tuple[AnyTensor, ...]):
-    if len(tensors) != 1:
-        raise ValueError("Tracing more than one tensor at a time is not supported.")
-    tensor = unbox_tensor(unshard(tensors[0]))
-    iree.turbine.ops.iree.trace_tensor(key, tensor)
+    # Skip tracing during export
+    if torch.compiler.is_compiling():
+        return
+    
+    for i, tensor in enumerate(tensors):
+        tensor = unbox_tensor(unshard(tensor))
+        iree.turbine.ops.iree.trace_tensor(f"{key}.{i}", tensor)
```

Comment:
The problem is that this will cause tensors trace in eager and IREE to have different keys.